templates/repo/issue/list.tmpl

```diff
@@ -171,6 +171,7 @@
 					{{else}}
 						<div class="ui red active basic button issue-action" data-action="close" data-url="{{$.RepoLink}}/issues/status" style="margin-left: auto">{{.locale.Tr "repo.issues.action_close"}}</div>
 					{{end}}
+					<div class="ui negative active button issue-action" data-action="delete" data-url="{{$.RepoLink}}/issues/status" style="margin-left: auto">{{.locale.Tr "repo.issues.delete"}}</div>
```

Comment:
In addition, why display a text when it's probably a better idea to use the `octicon-trash` SVG, first of all to minimize the user potential to missclick there, and secondly to improve the UX?